fisco-bcos/main/main.cpp

```diff
@@ -105,6 +106,15 @@ string initCommandLine(int argc, const char* argv[])
     return configPath;
 }
 
+void checkAndCall(const std::string& configPath, shared_ptr<Initializer> initializer)
+{
+    std::string moreGroupSignal = configPath + ".start_more_group";
```

Comment:
default is `./config.ini.start_more_group` ?
-c config.ini ==> config.ini..start_more_group ?

Reply:
modify to ./config.ini.append_group